README.md

```diff
@@ -1,2 +1,16 @@
 # on-skupack
 Maintain SKU specific package content such as templates and other static files.
+
+# Quick Start
+* Build SKU package
+
+```
+$ ./build-package.bash <sku_pack_directory> <subname>
+```
+
+* Register a new SKU with the package
+```
+$ curl -X POST --data-binary @sku_pack_directory_subname.tar.gz http://localhost:8080/api/common/skus/pack
```

Comment:
multipart/form-data should be used and the real location of the tarball should be referenced:
curl -X POST -F file=@tarballs/sku_pack_directory_subname.tar.gz localhost:8080/api/common/skus/pack